### ISSUE TITLE
Discord's own self bot apps

### ISSUE THREAD

**Issue by couleurm** [fr3fou/discord-mutual-friends#6]
Any plans to make something that's less in a gray area of the self botting ToS using new discord features?

**Comment by fr3fou** [comment]
yeah, I've been meaning to make something proper, but unfortunately Discord doesn't let you query the /users/@me/relationships endpoint through their OAuth API unless you go through their manual approval process.

**Comment by couleurm** [comment]
is this something that's out of reach for you?

**Comment by fr3fou** [comment]
as far as I know, going through this approval process is hard. they want to limit it to prevent abuse from bots, ig

**Comment by Landy005** [comment]
can this get you banned?
